test/unit/utils_test.js

```diff
@@ -94,24 +150,34 @@ describe('Utils', function() {
       expect(maxTime).to.eql(600);
       expect(maxTime - minTime).to.eql(600);
     });
-  });
 
-  describe("Utils.readChallengeTx", function() {
-    it('requires a non-muxed account', function() {
+    it("throws an error if a muxed account and memo is passed", function () {
+      let keypair = StellarSdk.Keypair.random();
+      const muxedAccount = new StellarSdk.MuxedAccount(
+        new StellarSdk.Account(
+          StellarSdk.Keypair.random().publicKey(), "-1"
```

Comment:
I know this is just test code, but negative sequence numbers can't exist, so passing `"0"` makes more sense here.

Reply:
I think we're using `-1` in other tests, not sure if there was a reason for it. I can use `0` here though.

Reply:
Hm okay, let's keep it consistent then. I can do a one-pass from -1 -> 0 in a separate PR.